app/src/main/java/io/horizontalsystems/bankwallet/modules/balance/BalanceFragment.kt

```diff
@@ -186,7 +187,7 @@ class CoinsAdapter(private val listener: Listener) : RecyclerView.Adapter<Recycl
         if (payloads.isEmpty()) {
             holder.bind(viewDelegate.getViewItem(position), expandedViewPosition == position)
         } else if (payloads.any { it is Boolean }) {
-            holder.bindPartial(expandedViewPosition == position)
+            holder.bindPartial(expandedViewPosition == position, viewDelegate.getViewItem(position).state is AdapterState.Syncing)
```

Comment:
It is better to store if a wallet is syncing in a holder when "bind()" is called and use it when "bindPartial()" is called.